migration_steps/initialise_environments/initialise_environments.sh

```diff
@@ -12,18 +12,7 @@ done
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-# envcheck - linked skeleton data should NOT be deleted on prod
-if [ "${ENVIRONMENT}" == "local" ] \
-  || [ "${ENVIRONMENT}" == "development" ] \
-  || [ "${ENVIRONMENT}" == "preproduction" ] \
-  || [ "${ENVIRONMENT}" == "preqa" ] \
-  || [ "${ENVIRONMENT}" == "qa" ]
-  then
-  python3 "${DIR}/delete_existing_data/app.py"
-else
-  echo "delete_existing_data should not run on ${ENVIRONMENT}"
-fi
-
+python3 "${DIR}/delete_existing_data/app.py"
```

Comment:
This makes me nervous

Reply:
I'd be nervous too if it was my script! 😄  On a serious note - we've had several people look at it and sign off on it + there's the assurance from Jack's count checks and UAT. Should be fine